autoload/verilog_systemverilog.vim

```diff
@@ -586,7 +586,12 @@ function! verilog_systemverilog#VerilogErrorFormat(...)
   endif
   if (l:tool == "ncverilog")
     " Based on https://github.com/vhda/verilog_systemverilog.vim/issues/88
-    set errorformat=%.%#:\ \*%t\\,%.%#\ \(%f\\,%l\|%c\):%m
+    set errorformat=%A%p\|,%Z%.%#:\ \*%t\\,%.%#\ \(%f\\,%l\|%c\):%m
```

Comment:
Why do you need the `%p`, given that column number is provided in the last message?